pymtl3/passes/sverilog/translation/structural/SVStructuralTranslatorL3.py

```diff
@@ -70,12 +69,11 @@ def gen_interface_array_decl( ifc_id, ifc_rtype, n_dim, c_n_dim, port_decls ):
       ret = []
       if not n_dim:
         id_ = ifc_id + c_n_dim
-        return map( lambda pdecl: pdecl.format( id_ = id_ ), port_decls )
+        return [pdecl.format( id_ = id_ ) for pdecl in port_decls]
       else:
-        return reduce( lambda res, l: res+l, map(
-          lambda idx: gen_interface_array_decl(
+        return reduce( lambda res, l: res+l, [gen_interface_array_decl(
```

Comment:
```python
return sum( [gen_interface_array_decl(
```

with the rest the same.

Reply:
TIL `sum(a_list_of_lists, [])` is a thing!